file-formats/auth/type/zif_aff_auth_v1.intf.abap

```diff
@@ -0,0 +1,108 @@
+INTERFACE zif_aff_auth_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">ABAP Dictionary</p>
+    "! ABAP dictionary
+    "! $required
+    BEGIN OF ty_abap_dictionary,
+      "! <p class="shorttext">Data Element</p>
+      "! Name of the data element
+      "! $required
+      dtel                           TYPE c         LENGTH 30,
+      "! <p class="shorttext">Domain</p>
+      "! Name of the domain
+      domname                        TYPE c         LENGTH 30,
```

Comment:
As far as I understood, the domain is the domain that is actually referenced in the referenced data element above. If this is the case, we should remove the domain name from the model.

Most probably, the same is also valid for the output length and the conversion routine, isn't it?